Source/Scene/ModelExperimental/ModelExperimentalPrimitive.js

```diff
@@ -118,8 +130,21 @@ function initialize(runtimePrimitive) {
     primitive.attributes
   );
 
+  var pointCloudShading;
+  if (ModelExperimentalType.is3DTiles(model.type)) {
+    pointCloudShading = model.content.tileset.pointCloudShading;
```

Comment:
Hm, no it would be better if the ModelExperimental3DTileContent forwarded the pointCloudShading so it's always `model.pointCloudShading`. This avoids checks like this in a couple places.

Reply:
> Hm, no it would be better if the ModelExperimental3DTileContent forwarded the pointCloudShading so it's always `model.pointCloudShading`. This avoids checks like this in a couple places.

Do you still plan on making this change?

Reply:
oh, looks like I missed a spot...